foo.py

```diff
@@ -23,6 +23,7 @@ def foo_two(z, x, y):
 def foo_three(a, g, j):
     """do foo three"""
     bat.do_a(a)
+    bat.do_g(g)
```

Comment:
gituhb review comment two

Reply:
one more test

Reply:
are we online

Reply:
are we online again

Reply:
**mike bayer** (zzzeek) wrote:

yes we are online

Reply:
ok no that wasn't quite online

Reply:
**mike bayer** (zzzeek) wrote:

NOW we are online